docs/rmis.md

```diff
@@ -0,0 +1,243 @@
+-   Setup and customizations require IT expert resources
+
+-   Ongoing costs for support and maintenance
+
+-   License costs based on time, user numbers, and support (if
+    proprietary)
+
+ *Potentially suitable for all cases if the project's / PEA's proce-
+ dures being digitalized / established are not too complex and do not
+ require too much customization.*
+
+#### Commercial off-the-shelf software (COTS)
+
+ Standard tools that can be used "as is" without the option to be
+ modified on software source code level.
+
+ \+ Available "out of the box" with basic functionality
+
+ \+ Suitable for simple project setups that do not require
+ customization
+
+## Interoperability Requirements
+
+ In which formats and structures should data be stored in the R / MIS to allow for an exchange with others relevant to the pro- ject / organization? In general, the R / MIS must fit into the already
+ existing IT architecture and with the existing IT knowledge of the PEA
+ and country. The following data standards should be consid- ered as
+ part of the setup of any R / MIS:
+
+-   Consider (project-specific) required file formats (e.g., CSV, XLS,
+    KML, JSON) or Application Programming Interfaces (API) for data
+    import and export
+-   KfW requires the export of geo-referenced project data (in KML or
+    XLS) according to a new template (see below) as part of its
+    regular reporting. This template also ensures compatibility of the
+    reporting with the International Aid Transparency Initiative (IATI)
+    Standard.
+-   The IATI standard is an XML-based set of rules and guidance for
+    publishing data of development and humanitarian organiza- tions to
+    improve their coordination, accountability, and effec- tiveness and
+    to increase the transparency of information on resources flowing
+    into developing countries. All BMZ-funded projects must provide
+    IATI-compatible data to KfW.
+-   Open Data Kit (ODK) is an XLS-based standard format for exchanging
+    data between R / MIS, mobile data collection, and other compatible
+    tools.
+-   Sector-specific open data standards also must be considered, if they
+    are applicable (e.g., EPA standard for environmental data;
+    [www.epa.gov/data-standards](http://www.epa.gov/data-standards)).
+
+## Legal Aspects
+
+ **Data Security:** If a desktop software system is used, data secu-
+ rity should be warranted by ensuring:
+
+ that the data are protected against unauthorized or accidental
+ destruction, accidental loss, technical defects, falsification, theft,
+ unlawful use, unauthorized modification or duplication, and other
+ forms of unauthorized access and use by technical means regular backup of data appropriate security of hardware
+
+ If the software is provided as web-based service (SaaS), the provider should be contractually obliged to take appropriate organizational and technical precautions to prevent disruptions to the
```

Comment:
see comment above. text in this line is very long, the lines below are quite short. Can it be made coherent?